Linear_cell_complex/include/CGAL/Linear_cell_complex/IO/VTK.h

```diff
@@ -60,7 +60,7 @@ namespace CGAL {
  * \return `true` if loading was successful, `false` otherwise
  */
 template <typename LCC, typename ScalarType = float>
-bool read_lcc_from_vtk(LCC& alcc,
+bool read_VTK(LCC& alcc,
```

Comment:
You need to use 2 different template type ScalarType1 and ScalarType2 because we can have different values associated with vertices and volumes. (same remark for all functions)